plugins/inputs/opcua/opcua_client.go

```diff
@@ -48,13 +49,24 @@ type OpcUA struct {
 }
 
 // OPCTag type
-type OPCTag struct {
-	Name           string `toml:"name"`
+type NodeSettings struct {
+	FieldName      string `toml:"field_name"`
 	Namespace      string `toml:"namespace"`
 	IdentifierType string `toml:"identifier_type"`
 	Identifier     string `toml:"identifier"`
-	DataType       string `toml:"data_type"`
-	Description    string `toml:"description"`
```

Comment:
removing these will make old configs fail. A concern?

Reply:
I added them back for compatibility but removed them from the readme because they were unnecessary and were never actually used.